homeassistant/util/color.py

```diff
@@ -8,7 +8,7 @@
 _LOGGER = logging.getLogger(__name__)
 
 HASS_COLOR_MAX = 500  # mireds (inverted)
-HASS_COLOR_MIN = 154
+HASS_COLOR_MIN = 111
```

Comment:
I don't have Hue lights, but one spec sheet I found said it supports 2000-6500K which matches 500-154 mired. So it seems that this change could break Hue.

The LIFX Color 1000 says 2500-9000K and the LIFX White 800 has 2700-6500K on the spec sheet.

Therefore, I don't think a global constant in Home Assistant can even work. It should be the individual device that reports its temperature range.